### PR TITLE
Expose name info of drawable node

### PR DESCRIPTION
**Pull request by JSUYA**
We need to expose the ketpath corresponding to the 'Fill' and 'Stroke' content.
The drawable receives the name and passes it to the node of capi.

### CODE REVIEW

File: src/vector/vdrawable.h [hunk 1]
```diff
@@ -54,6 +55,11 @@ class VDrawable {
     void preprocess(const VRect &clip);
     void applyDashOp();
     VRle rle();
+    void setName(const char *name)
+    {
+        if (name) mName = strdup(name);
```
Comment (hermet) [inline src/vector/vdrawable.h]: Guess this keypath is unique id, does it necessary copy names every frames?

Comment (JSUYA) [reply src/vector/vdrawable.h]: Here there is no frame info. The name value is set when the item's constructor is called.

**Comment by smohantty** [comment]
@JSUYA ,
who deletes the string passed to setName()?

**Comment by JSUYA** [comment]
> @JSUYA ,
> who deletes the string passed to setName()?

In VDrawable's destructor, it will deleted.(src/vector/vdrawable.cpp +37)
buy, if lottieitem set duplicate, leak may occur. I made a new patch.
https://github.com/Samsung/rlottie/pull/329

**Pull request merged**